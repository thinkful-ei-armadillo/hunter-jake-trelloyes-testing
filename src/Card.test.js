import React from 'react';
import ReactDOM from 'react-dom';
import Card from './card';
import renderer from 'react-test-renderer';

describe('<Card />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
})

describe('<Card />', () => {
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Card title = "first card" content = "lorem ipsum"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });
})