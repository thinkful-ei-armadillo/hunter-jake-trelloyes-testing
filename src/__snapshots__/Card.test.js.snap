// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders the UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
    first card
  </h3>
  <p>
    lorem ipsum
  </p>
</div>
`;
